src/services/zapper.js

```diff
@@ -0,0 +1,228 @@
+// @flow
+/*
+    Pillar Wallet: the personal data locker
+    Copyright (C) 2021 Stiftung Pillar Project
+
+    This program is free software; you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation; either version 2 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License along
+    with this program; if not, write to the Free Software Foundation, Inc.,
+    51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+*/
+/* eslint-disable i18next/no-literal-string */
+
+// utils
+import httpRequest from 'utils/httpRequest';
+import { reportErrorLog } from 'utils/common';
+
+// constants
+import { CHAIN } from 'models/Chain';
+import { PROTOCOLS } from 'constants/zapperServiceConstants';
+
+// types
+import type { Chain } from 'models/Chain';
+
+// does not change between envs
+const ZAPPER_CONFIG = {
+  API_URL: 'https://api.zapper.fi/v1',
+  API_KEY: '96e0cc51-a62e-42ca-acee-910ea7d2a241', // public
+};
+
+const requestConfig = {
+  timeout: 10000,
```

Comment:
Could we remove this timeout altogether, just incase we have a scenario where network conditions or Zapper server load slows down the response time. I'm always happy to wait _a bit_ longer to get my data - eventually.